lib/generator/exampleGenerator.ts

```diff
@@ -173,12 +173,18 @@ export default class Generator {
         return [];
       }
     } else {
+      const xMsExamples = specItem?.content?.["x-ms-examples"] || {};
+      const xMsExampleKeys = Object.getOwnPropertyNames(xMsExamples);
+      const title = xMsExampleKeys.length > 0 ? xMsExampleKeys[0] : "";
       example = {
-        title: specItem.content.summary
-          ? specItem.content.summary
-          : specItem.content.description
-          ? specItem.content.description
-          : operationId,
+        title:
+          title.length > 0
+            ? title
```

Comment:
Title should include the rule name, otherwise it will be duplicated. i.e. {title}_maximumSet and {title}_minimumSet